cirrus/server/tests/conftest.py

```diff
@@ -1,13 +1,15 @@
 from unittest import mock
 
 from fastapi.testclient import TestClient
+from mozilla_nimbus_shared import check_schema
 from pytest import fixture
 
 from cirrus.experiment_recipes import RemoteSettings
 from cirrus.feature_manifest import FeatureManifestLanguage
-from cirrus.main import app
+from cirrus.main import app, initialize_glean
 from cirrus.sdk import SDK
 from cirrus.settings import channel, context, fml_path
+from glean import testing
```

Comment:
Again, this is technically third-party. It should go with the second import group.

Reply:
I wonder why the linters aren't picking it up

Reply:
Yeah I've just been running `make cirrus_code_format` and letting it do its thing

Reply:
Its because there's a clean folder parallel to the tests folder. I just changed the name to ```gleann``` and the linter moved the import to the right spot. Not sure if that folder needs to be named "glean".

Reply:
I think I might adjust it to be telemetry?